packages/design-tokens/src/figma/tokens.json

```diff
@@ -379,15 +379,15 @@
       }
     },
     "fontWeights": {
-      "euclid-circular-b-0": {
+      "centra-no1-0": {
         "value": "Medium",
         "type": "fontWeights"
       },
-      "euclid-circular-b-1": {
+      "centra-no1-1": {
         "value": "Bold",
         "type": "fontWeights"
       },
-      "euclid-circular-b-2": {
+      "centra-no1-2": {
```

Comment:
Removed *Euclid* from the Figma JSON and replaced it with *Centra*. Not sure if Figma actively uses these, but we can use them for our tests. This change shouldn't have any other repercussions.